### PR TITLE
Restore the Docs

### PR DESCRIPTION
**Pull request by JonathonMisiewicz**
## Description
This PR fixes a bug accidentally introduced in #2190 that prevented docs updates.

1.4 please - a new release had better have docs that actually work.

## Questions
- [x] Some "sections" have a section line like the one I change in this PR, others don't. Local sphinx has a light blue color for the ones with the section line, dark blue for the ones without it. For standardization, should we just remove all of these...?

## Checklist
- [x] Sphinx appeased, locally

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

**Review by loriab — APPROVED** [review]
thanks. the light/dark blue have to do with how Sphinx computes header hierarchy. I had thought that the inconsistencies had to do with the symbols under the headers in reST (there is a preferred hierarchy), but maybe it's taking presence/absence of `.. sec:` markers into account too or instead. If it helps with standardization, probably every heading should have a `sec:` since otherwise one has to reference a section by full `Restarting the SCF`.

**Review by jturney — APPROVED** [review]

**Review by hokru — APPROVED** [review]

**Comment by JonathonMisiewicz** [comment]
I'll edit to give every section a header once I get to the office. If there are no objections, I'll assume this is still approved so long as docs still build locally.

**Comment by JonathonMisiewicz** [comment]
And, updated. This'll be good to go once tests pass. Sphinx build was clean, and the edited files looked good when viewed locally. This is a bigger change than I expected, so I'll leave it to somebody else to double-check my work and merge in.

The dark blue vs. light blue thing turned out not to be related to section headers, but to whether I had clicked on a specific subsection.

File: doc/sphinxman/source/scf.rst
Comment (loriab) [inline doc/sphinxman/source/scf.rst:243]:
```suggestion
.. _`sec:scfsymm`:
```

File: doc/sphinxman/source/scf.rst
Comment (loriab) [inline doc/sphinxman/source/scf.rst:575]:
```suggestion

.. _`sec:scfconv`:
```

File: doc/sphinxman/source/scf.rst
Comment (loriab) [inline doc/sphinxman/source/scf.rst:122]:
```suggestion

.. _`sec:scftheory`:
```
it was probably fine as-was since it built locally for you, but it's safer in Sphinx to be generous with blank lines

**Review by loriab — APPROVED** [review]
lgtm!

**Comment by JonathonMisiewicz** [comment]
Thanks. Clear to merge once tests re-pass.

**Comment by JonathonMisiewicz** [comment]
_sigh_ We have test fail because `adcc` broke. I'm holding this PR innocent. Something is probably wrong with `adcc` itself.

**Comment by loriab** [comment]
> sigh We have test fail because adcc broke. I'm holding this PR innocent. Something is probably wrong with adcc itself.

weird, it didn't install for that lane. adcc conda package hasn't changed in over a month. i'll try restarting that lane in case of fluke. otherwise, i'm guessing some non-qc dependency issue.

**Comment by JonathonMisiewicz** [comment]
Not a fluke. Note l. 93 of the configure build step reports that a suitable `adcc` could not be found, despite its inclusion in the build environment step. Sorry Lori, but I'm relying on you to solve this one. I have no idea how to isolate the dependency issue.

**Pull request merged**
